components/dashboard/src/app/AppRoutes.tsx

```diff
@@ -167,6 +158,7 @@ export const AppRoutes: FunctionComponent<AppRoutesProps> = ({ user, teams }) =>
                 {isLocalPreview() && <LocalPreviewAlert />}
                 <AppNotifications />
                 <Switch>
+                    <Route path="/new" exact component={CreateWorkspacePage} />
                     <Route path={projectsPathNew} exact component={NewProject} />
                     <Route path="/open" exact component={Open} />
```

Comment:
**suggestion**: Shall we drop this route now as it seems duplicate and broken? See [relevant discussion](https://gitpod.slack.com/archives/C01KPEPQYE6/p1673523784907079?thread_ts=1672836161.790069&cid=C01KPEPQYE6) (internal).